Automation/Mobile/Troubleshooting Appium.md

```diff
@@ -20,36 +20,35 @@ NOTE:
 
 Appium Server and Appium Desktop are installed in two different places.
 
-Appium Server: `/usr/local/lib/node_modules/appium/node_modules/appium-webdriveragent`
+Appium Server: 
 
-Appium Desktop: `/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-webdriveragent`
+`/usr/local/lib/node_modules/appium/node_modules/appium-webdriveragent`
 
+Appium Desktop: 
 
-## Running tests
-
-Appium runs tests alphabetically. It will first go through test files by name in alphabetical order. Then it will again go alphabetically through the tests inside each test file/class.
-
-One way to override that is to name your tests prefixed with numbers, something like:
-
-- test_001 (or test_001_001)
-- test_001_002 
-- ...
-
-If your tests are independent of each other, then you might not have to worry about the test order. In some cases, you might need to link a few tests together and in that case, you will have to keep this in mind.
```

Comment:
We don't need this section anymore in this article? 😄

Reply:
Discussed on Slack